build_install_pythons.sh

```diff
@@ -1,25 +1,24 @@
 #!/bin/bash
-# Install Pythons 2.7 3.4 3.5 3.6 3.7 3.8rc1 and matching pips
+# Install Pythons and matching pips
 set -ex
 
-echo "deb http://ppa.launchpad.net/deadsnakes/ppa/ubuntu trusty main" > /etc/apt/sources.list.d/deadsnakes.list
+echo "deb http://ppa.launchpad.net/deadsnakes/ppa/ubuntu bionic main" > /etc/apt/sources.list.d/deadsnakes.list
 apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 6A755776
 apt-get update
 apt-get install -y wget
 PIP_ROOT_URL="https://bootstrap.pypa.io"
 wget $PIP_ROOT_URL/get-pip.py
-for pyver in 3.4 3.5 3.6 2.7 2.6 3.3 ; do
+apt-get install -y python3-distutils
+for pyver in 3.5 3.6 3.7 3.8 2.7; do
```

Comment:
It turns out deadsnakes has everything we need. Dropping 3.4, 3.3. Do any projects still need them?

Reply:
I don't think so.  Multibuild doesn't support those versions, as you know.